bin/tictactoe

```diff
@@ -1,3 +1,36 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
+input = ""
+
+while input != "exit"
+  puts "How many humans will be playing? [0,1,2]: "
+  player_num = gets.strip.to_i
+
+  if player_num.to_i == 0
+    player1 = Player::Computer.new("X")
+    player2 = Player::Computer.new("O")
+  elsif player_num.to_i == 1
+    puts "Who is going first? You or the computer? [me][computer]"
+    first = gets.strip
+    if first == "me"
+      player1 = Player::Human.new("X")
+      player2 = Player::Computer.new("O")
+    else
+      player1 = Player::Computer.new("X")
+      player2 = Player::Human.new("O")      
+    end
+  elsif player_num.to_i == 2
+      player1 = Player::Human.new("X")
+      player2 = Player::Human.new("O")
+  end
+
+  game = Game.new(player1, player2)
+  game.board.display
+  game.play
+
+  puts "Would you like to play again or exit? [play again][exit]"
+  input = gets.strip
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.